netebpfext/net_ebpf_ext_sock_ops.c

```diff
@@ -66,7 +66,7 @@ typedef struct _net_ebpf_extension_sock_ops_wfp_filter_context
 } net_ebpf_extension_sock_ops_wfp_filter_context_t;
 
 //
-// SOCK_OPS Global helper function implementation.
+// SOCK_OPS Global helper function implementations.
```

Comment:
Why this change since there's only one in line 93?

Reply:
will change it back